### PR TITLE
[turnstile] Add changelog

### PR DESCRIPTION
**Pull request by Le0Developer**
Adds changelog for new documented features/things introduced after open beta started.

Dates are when the feature was documented, not when it was added. (eg turnstile.remove was added a week earlier but not documented)

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]

File: content/turnstile/changelog.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Changelog
+
+## 2022-10-13
+
+- Added validation for action: `/^[a-z0-9_-]{0,32}$/i`
+- Added validation for cData: `/^[a-z0-9_-]{0,255}$/i`
+
+## 2022-10-11
+
+- Added `turnstile.remove`
```
Comment (kodster28) [inline content/turnstile/changelog.md]:
```suggestion
- Added [`turnstile.remove`](/turnstile/get-started/client-side-rendering/#remove-a-widget)
```

**Comment by kodster28** [comment]
@Le0Developer, you have any issues with me merging this, #6291, or #6292?

Seems like #6346 should wait since it's not live yet (only in staging). Or has it also graduated to production?

**Comment by Le0Developer** [comment]
@kodster28 This and #6291 are ready to merge. #6292 & #6346 are still in staging.

**Comment by Le0Developer** [comment]
Update: #6292 is now in production.

**Pull request merged**